source/schema/export.txt

```diff
@@ -24,156 +24,95 @@ If you have not already done so, analyze your schema:
    subset of documents from your collection. To learn more about 
    sampling, see :ref:`Sampling <compass-sampling>`.
 
-Once your schema has been analyzed, export your schema:
+Once your schema has been analyzed, export your schema. 
 
-1. In the top menu bar, click :guilabel:`Collection`.
+1. Click :guilabel:`Export Schema`.
 
-#. From the dropdown, click :guilabel:`Share Schema as JSON`.
+   .. figure:: /images/compass/export-schema-button.png
+      :alt: Image showing Export Schema button
 
-.. figure:: /images/compass/share-schema-json.png
-   :alt: Image showing Collection dropdown
+   The :guilabel:`Export Schema` button appears in the top-left corner
+   of the viewport.
 
-Your schema is copied to your clipboard as a |json| object.
+#. In the **Export JSON Schema** modal, select the format to export to.
```

Comment:
[nit] I believe this should be :guilabel:`Export JSON Schema`!

Reply:
Updated.